### ISSUE TITLE
Cannot view the app on AA

### ISSUE THREAD

**Issue by kyokusei84** [martoreto/aauto-sdk-demo#8]
Sadly on my car I cannot see the demo option after build it. I tried to change the package name too, but didn't work. On PC, with the same smarthpone and app, with AA emulator works perfectly. How is it possible?
I downgraded AA to 2.9 version, but I cannot downgrade the google play services.

**Comment by mrmstn** [comment]
The problem is, google stopt all non-playstore AA Apps from working. As you already mentioned, one workaround would be, to downgrade your playservice - the other one would be to publish the demo app in the playstor (yeah :sweat_smile:  - I've no idea what google thought about that.)

Anyway - whats the issue with downgrading your play-service?

**Comment by lucav90** [comment]
Hi,

I have the same problem with other custom android auto app. There Is a solution? On emulator works but on car not. 
The solution is also publish app to google play or downgrade playservices? 
Witch version of google play services works correctly?

Regards,

**Comment by andrea1984** [comment]
You have to enable the android auto developer mode and check the unknown sources option.

**Comment by lucav90** [comment]
Yes android auto developer mode is on, unknow resources option Is on, but the problem Is the same.
With android auto emulator i view app on list and i can open It.
On real android auto in my car i havent the app in the list so i can't open It.
What Is the solution?

Regards

**Comment by kyokusei84** [comment]
Is it not possibile to build a custom app without Google permission. You can only build notification for AA or a custom audio player for AA, that's how I did. I made a custom player, using the playlist as custom buttons for making the needed actions, and using notifications as response.

**Comment by lucav90** [comment]
Hi, you have a sample app of this so i can try it if works?
you have only the radio player on this app or other functions?

regards,

**Comment by andrea1984** [comment]
> Hi, you have a sample app of this so i can try it if works?
> you have only the radio player on this app or other functions?
> 
> regards,

Open Android studio, create a new project and choose Media Service from Automotive tab.

**Comment by lucav90** [comment]
Thanks. But Is possible to use this workaround of messaging AA app to display data or android acrivity?

**Comment by andrea1984** [comment]
No you can't. You can only show (short) data via notifications.

**Comment by lucav90** [comment]
Hi, with root and AA-Phenotype-Patcher is possibile to view custom app on real android auto?

**Comment by jilleb** [comment]
Yes.
